### ISSUE TITLE
iOS 11 Support

### ISSUE THREAD

**Issue by sandstrom** [phonegap/phonegap-plugin-image-capture#25]
The current implementation on iOS is somewhat inferior compared to Android. My understand is that this is because Safari has less support for WebRTC and Media Streaming.

But it seems like Safari 11 / iOS 11 has improved in this area. Will this enable you to bring iOS to parity with Android?

**Sources**
https://webkit.org/blog/7726/announcing-webrtc-and-media-capture/
https://developer.apple.com/library/content/releasenotes/General/WhatsNewInSafari/Safari_11_0/Safari_11_0.html

**Comment by macdonst** [comment]
@sandstrom yes, this is certainly something we are looking into. We are busy finishing up the media recorder plugin but we will circle back to this.

**Comment by macdonst** [comment]
See issue https://github.com/phonegap/phonegap-plugin-media-stream/issues/5

**Comment by macdonst** [comment]
@maverickmishra possibly we can skip the native implementation of this plugin on iOS 11 as well as we could use JS to capture the image. Here is a good gist:

https://gist.github.com/wesbos/ee0ad57f2ed710bbc2235aa9cd31a25a

**Comment by maverickmishra** [comment]
navigator.getMediaDevices is not available for WKWebView and UIWebView in iOS 11. It is only available for Safari

**Comment by sandstrom** [comment]
This radar entry may be relevant: http://www.openradar.me/33571214
